cluster/manifests/postgres-operator/credentials.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "zalando.org/v1"
+kind: PlatformCredentialsSet
+metadata:
+  name: postgresql-operator
+spec:
+  application: postgresql-operator
+  tokens:
+    read-only:
+      privileges:
+    cluster-registry-rw:
```

Comment:
We neither read nor write it. I have no idea why it is there in the first place :-)